scrape_dbPedia.js

```diff
@@ -0,0 +1,98 @@
+const http = require('http');
+var express = require('express');
+var fs = require('fs');
+var request = require('request');
+var cheerio = require('cheerio');
+var app     = express();
+var sleep = require('system-sleep');
+var async = require('async');
+const cmp = require('comparejs');
+
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World\n');
+});
+
+  server.listen(port, hostname, () => {
+  console.log(`Server running at http://${hostname}:${port}/`);
+
+  var elemNum = 0;
+
+  var newObj = {
+    table: []
+  };
+  fs.readFile('myjsonfile2.json', 'utf-8', function(err, array) {
```

Comment:
[Here](https://github.com/MusicConnectionMachine/StructuredData/pull/12/files#diff-815b15a3748d5c262f4563790a730a0dR15) you use ES2015/ES6 syntax to write a function callback (`(param) ==> { //doSomething }`), while in this line you use ES5 (`function(param){ //doSomething }`). Stick to one standard, it's more readable :)